cli/determined_cli/trial.py

```diff
@@ -18,46 +17,26 @@
 
 @authentication_required
 def describe_trial(args: Namespace) -> None:
-    q = api.GraphQLQuery(args.master)
-    trial = q.op.trials_by_pk(id=args.trial_id)
-    trial.end_time()
-    trial.experiment_id()
-    trial.hparams()
-    trial.start_time()
-    trial.state()
-
-    steps = trial.steps(order_by=[gql.steps_order_by(id=gql.order_by.asc)])
-    steps.metrics()
-    steps.id()
-    steps.state()
-    steps.start_time()
-    steps.end_time()
-
-    checkpoint_gql = steps.checkpoint()
-    checkpoint_gql.state()
-    checkpoint_gql.uuid()
-
-    validation = steps.validation()
-    validation.state()
-    validation.metrics()
+    if args.metrics:
+        r = api.get(args.master, "trials/{}/metrics".format(args.trial_id))
+    else:
+        r = api.get(args.master, "trials/{}".format(args.trial_id))
 
-    resp = q.send()
+    trial = r.json()
 
     if args.json:
-        print(json.dumps(resp.trials_by_pk.__to_json_value__(), indent=4))
+        print(json.dumps(trial, indent=4))
         return
 
-    trial = resp.trials_by_pk
-
     # Print information about the trial itself.
     headers = ["Experiment ID", "State", "H-Params", "Start Time", "End Time"]
     values = [
         [
-            trial.experiment_id,
-            trial.state,
-            json.dumps(trial.hparams, indent=4),
-            render.format_time(trial.start_time),
-            render.format_time(trial.end_time),
+            trial["experiment_id"],
```

Comment:
Suggestion: I used `zip` to place the header names next to the corresponding values here https://github.com/determined-ai/determined/blob/77e045c31909e0c592fba1bf359123ee16f0c531/cli/determined_cli/checkpoint.py#L37-L52.

 I found it easier to reason able the table view and add/remove columns.

Reply:
Nice suggestion, it looks way better